controllers/onboardingExtension.ts

```
@@ -48,23 +48,25 @@ const requestModel = firestore.collection("requests");
+    res: OnboardingExtensionResponse)
     : Promise<OnboardingExtensionResponse> => {
 
     try {
 
         const data = req.body as CreateOnboardingExtensionBody;
-        const {user, userExists} = await fetchUser({discordId: data.userId});
-        
-        if(!userExists) {
+        const { user, userExists } = await fetchUser({ discordId: data.userId });
+        const { dev } = req.query;
+        const isDev = dev === "true" ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)